Makefile

```diff
@@ -32,6 +32,11 @@ export GOPROXY
 # Active module mode, as we use go modules to manage dependencies
 export GO111MODULE=on
 
+# Kubebuilder.
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.23.3
```

Comment:
@evanfreed I think the test failures are due to the new version, I opened a PR to simply bump the version and it's failing with the same errors: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/2418/pull-cluster-api-provider-azure-test/1540081111491678208

Let's just keep the version the same for now (1.20.2) and we can figure out what needs to change in our tests to accomodate the version bump separately.

Reply:
👍 sorry for the delay, I've had some vacation recently.  Updated based on your comment in the latest push.